pandas/core/indexes/datetimes.py

```diff
@@ -1964,18 +1967,33 @@ def tz_localize(self, tz, ambiguous='raise', errors='raise'):
 
             .. versionadded:: 0.19.0
 
-        infer_dst : boolean, default False
-            .. deprecated:: 0.15.0
-               Attempt to infer fall dst-transition hours based on order
 
         Returns
         -------
         localized : DatetimeIndex
 
+	Examples
+    	--------
```

Comment:
same comment here about the indentation (it seems you are mixing tabs and spaces. It is best to configure your editor to introduce 4 spaces when you do tab)